projects/storefrontlib/src/lib/cms/services/cms-mapping.service.ts

```diff
@@ -18,57 +12,25 @@ export class CmsMappingService {
     @Inject(PLATFORM_ID) private platformId: Object
   ) {}
 
-  /**
-   * The "JspIncludeComponent" and "FlexCmsComponent" are types of CmsComponent that behave
-   * as a placeholder component (with no specific data provided).
-   *
-   * While it's not very clean solution, we interpret the "uid" of the "JspIncludeComponent" and "flextype" of "FlexCmsComponent"
-   * as a component type and thanks to that we map it onto the implementation of the Angular (or web) component.
-   *
-   * CAUTION: The mapped type should not be used for SmartEdit bindings.
-   */
-  getMappedType(component: ContentSlotComponentData): string {
-    switch (component.typeCode) {
-      case JSP_INCLUDE_CMS_COMPONENT_TYPE:
-        return component.uid;
-      case CMS_FLEX_COMPONENT_TYPE:
-        return component.flexType;
-      default:
-        return component.typeCode;
-    }
-  }
-
-  isMappedTypeEnabled(mappedType: string): boolean {
+  isFlexTypeEnabled(flexType: string): boolean {
```

Comment:
We operate in cms mapping only on flex types. I think we can drop `flex` here and leave `isComponentEnabled`. It will be consistent with other methods of this service.

Reply:
👍